src/ASM/ASMs3Dmx.C

```diff
@@ -1251,9 +1251,6 @@ void ASMs3Dmx::generateThreadGroups (const Integrand& integrand, bool silence,
 
 void ASMs3Dmx::generateThreadGroups (char lIndex, bool silence, bool)
 {
-#ifdef USE_OPENMP
-  omp_set_num_threads(1);
-#endif
```

Comment:
But then you could just remove the overriding and just use the parent class method, since it be just the same anyway?

Reply:
yeah, right you are.